src/test/java/io/confluent/connect/jdbc/sink/PreparedStatementBinderTest.java

```diff
@@ -108,57 +130,59 @@ public void bindRecord() throws SQLException {
   @Test
   public void bindFieldPrimitiveValues() throws SQLException {
     int index = ThreadLocalRandom.current().nextInt();
-    verifyBindField(++index, Schema.Type.INT8, (byte) 42).setByte(index, (byte) 42);
-    verifyBindField(++index, Schema.Type.INT16, (short) 42).setShort(index, (short) 42);
-    verifyBindField(++index, Schema.Type.INT32, 42).setInt(index, 42);
-    verifyBindField(++index, Schema.Type.INT64, 42L).setLong(index, 42L);
-    verifyBindField(++index, Schema.Type.BOOLEAN, false).setBoolean(index, false);
-    verifyBindField(++index, Schema.Type.BOOLEAN, true).setBoolean(index, true);
-    verifyBindField(++index, Schema.Type.FLOAT32, -42f).setFloat(index, -42f);
-    verifyBindField(++index, Schema.Type.FLOAT64, 42d).setDouble(index, 42d);
-    verifyBindField(++index, Schema.Type.BYTES, new byte[]{42}).setBytes(index, new byte[]{42});
-    verifyBindField(++index, Schema.Type.BYTES, ByteBuffer.wrap(new byte[]{42})).setBytes(index, new byte[]{42});
-    verifyBindField(++index, Schema.Type.STRING, "yep").setString(index, "yep");
+    verifyBindField(++index, Schema.INT8_SCHEMA, (byte) 42).setByte(index, (byte) 42);
+    verifyBindField(++index, Schema.INT16_SCHEMA, (short) 42).setShort(index, (short) 42);
+    verifyBindField(++index, Schema.INT32_SCHEMA, 42).setInt(index, 42);
+    verifyBindField(++index, Schema.INT64_SCHEMA, 42L).setLong(index, 42L);
+    verifyBindField(++index, Schema.BOOLEAN_SCHEMA, false).setBoolean(index, false);
+    verifyBindField(++index, Schema.BOOLEAN_SCHEMA, true).setBoolean(index, true);
+    verifyBindField(++index, Schema.FLOAT32_SCHEMA, -42f).setFloat(index, -42f);
+    verifyBindField(++index, Schema.FLOAT64_SCHEMA, 42d).setDouble(index, 42d);
+    verifyBindField(++index, Schema.BYTES_SCHEMA, new byte[]{42}).setBytes(index, new byte[]{42});
+    verifyBindField(++index, Schema.BYTES_SCHEMA, ByteBuffer.wrap(new byte[]{42})).setBytes(index, new byte[]{42});
+    verifyBindField(++index, Schema.STRING_SCHEMA, "yep").setString(index, "yep");
```

Comment:
missing `verifyBindField()` calls for`decimal`, `date`, `time`, and `timestamp` here